src/app.js

```diff
@@ -0,0 +1,35 @@
+import tasks from './tasks.js';
+
+export function addTask(tasks, description) {
+  const task = {
+    description,
+    completed: false,
+    index: tasks.length + 1,
+  };
+  tasks.push(task);
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
+
+export function editTask(index) {
+  const taskItem = document.querySelector(`#todo-list li:nth-child(${index + 1})`);
+  const descriptionElement = taskItem.querySelector('p');
+  const editInput = document.createElement('input');
+  editInput.type = 'text';
+  editInput.value = descriptionElement.innerText;
+  editInput.addEventListener('blur', () => {
+    tasks[index].description = editInput.value;
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+  });
+  descriptionElement.replaceWith(editInput);
+  editInput.focus();
+}
```

Comment:
- [ ] [OPTIONAL]
I noticed you implemented a click event for **editing/updating** added tasks, however, I'm unable to edit added task from the User Interface after several click attempts.  Please ensure this feature is working properly on the User Interface, else kindly add instruction in the ReadMe on how to achieve the same.

Reply:
Thank you for pointing this out, I will check that to see!

Reply:
Fixed, welcome to have a check on that again ><